.travis.yml

```diff
@@ -24,7 +21,9 @@ addons:
 
 before_install:
   - gem install rake
-  - gem install minitest stringex
-  - gem install rouge
+  - rake dev:gemspec
+  - rake dev:gemfile
 
-script: ruby -Ilib:test test/test_*
+script: bundle exec rake dev:profile_memory
```

Comment:
This project doesn't use bundler since it is not necessary. And Travis should run the tests, not profile memory (even if that may run some tests).

Reply:
> And Travis should run the tests, not profile memory (even if that may run some tests).

:smiley: I figured since Travis wasn't running on the project currently, you didn't want automated testing.. :stuck_out_tongue_winking_eye:

Reply:
Travis is running but tidy has a problem with one test... Locally everything is fine, so I don't really know what Travis is up to. I have updated the code to produce the error messages from tidy, will see if that helps.